AUTHORS.rst

```diff
@@ -0,0 +1,30 @@
+=======
+Credits
+=======
+
+Autoprotocol-Python is currently maintained by members of Strateos Inc.
+
+Here is a list of active contributors from Strateos:
+
+- Vanessa Biggers - `polarpine <https://github.com/polarpine>`_ - vanessa.biggers@strateos.com
```

Comment:
any thoughts on order here? not sure if there's any real order at the moment (which is fine). we could do something like:
 - tenure working with the project
 - alphabetical
 - ???

Reply:
ah, historically it was following last name alphabetical ordering. Was just following this along, but happy to change it

Reply:
ah gotcha! missed that it was last name alphabetical, thanks!